### PR TITLE
conda-gitenv recipe

### PR DESCRIPTION
**Pull request by bjlittle**

### CODE REVIEW

File: recipes/conda_gitenv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+    - gitpython
+    - pyyaml
+    - conda >=4.1.0
+    - conda-build-all
```
Comment (jakirkham) [inline recipes/conda_gitenv/meta.yaml]: So `conda-build` version `2.0.9` and `conda` version `4.1.x` are not compatibility due to a mix-up. Should be fixed in the next release. Would add an exclusion for `conda-build` version `2.0.9` specifically.

Comment (bjlittle) [inline recipes/conda_gitenv/meta.yaml]: Thanks @jakirkham ... I've just pinned `conda-build ==2.0.6` ... should that do the trick?

Comment (jakirkham) [inline recipes/conda_gitenv/meta.yaml]: Also would raise an issue in `conda-build-all` along the same lines as it should be doing this.

Sorry it's all kind of a mess as the solver can find ways around these pinnings. 😩

Comment (bjlittle) [inline recipes/conda_gitenv/meta.yaml]: Thanks, will do!

File: recipes/conda_gitenv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  commands:
+    - conda gitenv --help
+    - conda gitenv resolve --help
+    - conda gitenv autotag --help
+    - conda gitenv autolabel --help
+    - conda gitenv deploy --help
+
+about:
+  home: https://github.com/SciTools/conda-gitenv
+  license: BSD 3-Clause
```
Comment (jakirkham) [inline recipes/conda_gitenv/meta.yaml]: Could we please add `license_file`? Also optionally can add `license_family`.

**Comment by bjlittle** [comment]
@jakirkham @ocefpaf This looks good to go :wink:

File: recipes/conda_gitenv/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: conda-gitenv
```
Comment (jakirkham) [inline recipes/conda_gitenv/meta.yaml]: So the directory name and recipe name don't match. Hence this has failed to convert. Should it be a `-` or a `_`? Could you please send a follow up PR to fix this, @bjlittle ?

Comment (bjlittle) [inline recipes/conda_gitenv/meta.yaml]: @jakirkham Apologies, will do. 

Interesting. Would it be easy to have a simple automated check on the staged-recipes PR to detect this type of error earlier, before failing post merge?

I'll push a fresh PR with the correction, thanks.

**Pull request merged**